client/src/utils/dayjs.ts

```diff
@@ -0,0 +1,8 @@
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import utc from 'dayjs/plugin/utc';
+
+dayjs.extend(relativeTime);
+dayjs.extend(utc);
+
+export default dayjs;
```

Comment:
We should avoid exporting it to prevent picking it as a reference when using it in other files